build-system/compile/bundles.config.extensions.json

```diff
@@ -152,10 +152,19 @@
   {"name": "amp-facebook-page", "version": "0.1", "latestVersion": "0.1"},
   {
     "name": "amp-fit-text",
-    "version": ["0.1", "1.0"],
+    "version": "0.1",
     "latestVersion": "0.1",
     "options": {"hasCss": true}
   },
+  {
+    "name": "amp-fit-text",
+    "version": "1.0",
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true,
+      "npm": {"binaries": ["component.js"], "external": ["preact"]}
```

Comment:
What would CE binary look like here? Would it make sense for us to do:

```
npm: {preact: true, ce: true}
```

and hide the details where we process these keys?

Reply:
That's one possibility, but my intent here was to allow any file name without any other difficulty. Eg, maybe Accordion wants to export both the root `Accordion` and the sub `Secction` components using two separate files.

Reply:
And what about a React binary? The same `Accordion` and sub `Section` would then be relevant for React too, right?

Reply:
Yup. The exact package file names are decided in the `package.json`, so you could rename it anything. I haven't quite figured out how to do the React mapping yet. Maybe it'd be best to list the remapping here?

```js
{
  binaries: [
    {
      entryPoint: 'component.js',
      out: 'component.js',
      external: ['preact'],
    },
    {
      entryPoint: 'component.js',
      out: 'component-react.js',
      external: ['react'],
      remap: { 'preact': 'react' },
    }
  ]
}

Reply:
Sure. If it works. One thing I'd note - perhaps then we should also do `out: 'component-preact.js'`. Because, if anything, I think `-react.js` is more canonical.

Reply:
Done.